ci/garbage_collection/repo_cleanups/remove_stale_rpm.sh

```diff
@@ -0,0 +1,229 @@
+#!/bin/bash 
```

Comment:
This file still has the old code and isn't called anywhere. I guess you forgot to run `git rm`?